scripts/utils/migration_scripts/collectionsMigration/Makefile

```diff
@@ -0,0 +1,49 @@
+# see main.go for all env var options/defaults
+
+SHELL=bash
+APP=collections-migration
+
+ENV?=sandbox
+TIMEOUT?=30s
+
+host_num?=publishing_mount 1
+host_bin=bin-$(APP)
+
+GOOS?=$(shell go env GOOS)
+GOARCH?=$(shell go env GOARCH)
+
+BUILD=build
+BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
+BUILD_SCRIPT=$(BUILD_ARCH)/$(APP).sh
+
+.PHONY: all
+all: clean
+	make GOOS=linux build deploy
```

Comment:
Why is `make clean` setting `GOOS=linux`? Surely the os should be `darwin` when run locally. Only deployments to AWS should be `linux`. 
On a related note, it's not a good idea to set the `GOARCH` to the local architecture especially as M1 macs become more prevalent. In those cases the arch will become `arm64` rather than the `amd64` required by our current AWS deployments.

Reply:
added local running instructions and comment that the make file is about running remotely